hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -2317,28 +2461,42 @@ private void listStatusFindKeyInTableCache(
           throws IOException {
     Preconditions.checkNotNull(args, "Key args can not be null");
 
+    String volumeName = args.getVolumeName();
+    String bucketName = args.getBucketName();
+    String keyName = args.getKeyName();
+    boolean sortAddress = args.getSortDatanodes();
+    return listOzoneFileStatusesV1(recursive, startKey, 0, numEntries,
+            clientAddress, volumeName, bucketName, keyName, sortAddress,
+            false);
+  }
+
+  @NotNull
+  protected List<OzoneFileStatus> listOzoneFileStatusesV1(boolean recursive,
+      String startKey, int countEntries, long numEntries, String clientAddress,
+      String volumeName, String bucketName, String keyName,
+      boolean sortAddress, boolean listKeys) throws IOException {
```

Comment:
Can we update this as a private method, I see this is privately used in this class.

Reply:
Sure.